frontend/public/components/monitoring/alert-manager-yaml-editor.tsx

```diff
@@ -21,15 +20,11 @@ const EditAlertmanagerYAML = (props) => (
 
 const AlertManagerYAMLEditor: React.FC<AlertManagerYAMLEditorProps> = ({ obj }) => {
   const secret: K8sResourceKind = obj;
-  const encodedAlertManagerYaml = _.get(secret, ['data', 'alertmanager.yaml']);
-  const initErrorMsg = _.isEmpty(encodedAlertManagerYaml)
-    ? 'Error: alertmanager.yaml not found in Secret "alertmanager-main", in namespace "openshift-monitoring"'
-    : null;
-
-  const [errorMsg, setErrorMsg] = React.useState(initErrorMsg);
-  const [successMsg, setSuccessMsg] = React.useState();
-  const alertManagerYamlStr = !_.isEmpty(encodedAlertManagerYaml)
-    ? Base64.decode(encodedAlertManagerYaml)
+  const [errorMsg, setErrorMsg] = React.useState('');
+  const [loadErrorMsg, setloadErrorMsg] = React.useState('');
+  const [successMsg, setSuccessMsg] = React.useState('');
+  const alertManagerYamlStr = _.isEmpty(loadErrorMsg)
```

Comment:
I guess you could drop the `_.isEmpty(loadErrorMsg)` check if you move this after the `if (loadErrorMsg) { ... }` block.

Reply:
fixed